test/testbase.py

```diff
@@ -154,6 +154,12 @@ async def setup():
                 )
                 formed_connection.project = cls.project_id
 
+                # # Convert the relative path to an absolute path
```

Comment:
Do you want to keep this commented section?

Reply:
Was thinking maybe we may need to use this in another test class but I should probably remove it for now!